exercise_1.rb

```diff
@@ -13,6 +13,7 @@
   end
 end
 
-biggest = main_array.map(&:sum).max
+sum_arrays= main_array.map(&:sum)
+top_three= sum_arrays.sort.reverse.take(3)
 
-puts acum.inspect
```

Comment:
This is pretty simple so there isn't too much to comment one code wise. One this is to be mindful of variable names (`main_array`). Generally when writing code for exercises variable names are not important, but anything that others will be reading, it is always best to fall to the side of readability.